llarp/handlers/exit.cpp

```diff
@@ -51,9 +51,9 @@ namespace llarp
       util::StatusObject exitsObj{};
       for(const auto &item : m_ActiveExits)
       {
-        exitsObj.Put(item.first.ToHex(), item.second->ExtractStatus());
+        exitsObj[item.first.ToHex()] = item.second->ExtractStatus();
```

Comment:
while you're here it ideally should be `ToString()` not `ToHex()` so it reports the snode address